swap/htlc_test.go

```diff
@@ -319,3 +322,230 @@ func TestHtlcV2(t *testing.T) {
 		})
 	}
 }
+
+/*
+	CLAIM PATH
+
+	<reciever_key> OP_CHECKSIGVERIFY OP_SIZE 20 OP_EQUALVERIFY OP_RIPEMD160 <hash> OP_EQUALVERIFY 1 OP_CHECKSEQUENCEVERIFY
+*/
+func createClaimPathLeaf(t *testing.T, recieverHtlcKey [32]byte, swapHash lntypes.Hash) (txscript.TapLeaf, []byte) {
```

Comment:
Rather than storing `claimScript`/`timeoutScript` in `HtlcScriptV3`, move these functions into `htlc.go` and then we can just recreate them using our receiver/sender key and cltv when we need these scripts?